mx.truffle/suite.py

```diff
@@ -364,6 +364,21 @@
        "workingSets" : "Truffle,SimpleLanguage,Tools",
        "license" : "UPL",
      },
+
+    "com.oracle.truffle.examples" : {
+       "subDir" : "truffle",
+       "sourceDirs" : ["src"],
+       "dependencies" : [
+                        "TRUFFLE_API",
+                        ],
+       "annotationProcessors" : ["TRUFFLE_DSL_PROCESSOR"],
+       "checkstyle" : "com.oracle.truffle.sl",
+       "javaCompliance" : "1.8",
```

Comment:
Currently Truffle default compliance is 1.7 - it shouldn't be that big problem - just replace `Suplier` with `java.util.concurrent.Callable`.